packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -130,7 +130,7 @@ export function evaluateLazySchema<T extends TreeNodeSchema>(value: LazyItem<T>)
 type ExtractItemType<Item extends LazyItem> = Item extends () => infer Result ? Result : Item;
 
 // @alpha
-export function extractPersistedSchema(schema: ImplicitFieldSchema): JsonCompatible;
+export function extractPersistedSchema(schema: ImplicitFieldSchema, version: SchemaFormatVersion): JsonCompatible;
```

Comment:
Like exportCompressed, this should probably take in an options object so we can provide more settings later without adding lots of arguments, and take in the version as FluidClientVersion, which is already exposed as alpha  `options: {
        oldestCompatibleClient: FluidClientVersion;
    }`
    
    Look at /examples/apps/tree-cli-app/src/utils.ts (who's build you broke with this API change) for an example usage. It already uses exportCompressed with FluidClientVersion and calls extractPersistedSchema: passing in the same FluidClientVersion there would be nicer and clean compared to taking in a different version specific for schema stuff.